### ISSUE TITLE
Feature Request: Use front facing screen as a light source 

### ISSUE THREAD

**Issue by rogery555** [gopro/labs#76]
I would love the option be be able to set the front facing screen to all white so it could act as a light source to illuminate in dark close range filming situations where there are not other lighting options.  Although not very powerful ever bit of light can be helpful,  even now just displaying the video on the front screen does add some illumination so the ability to have it all white would be very nice!

**Comment by dnewman-gpsw** [comment]
Interesting, certainly doable. Although I do wonder if such a small and low light source, would do much for luminating a scene, but it could do something for the users face, adding a glint to the eye. Maybe for some vlogging ghost hunting videos it would be enough.

**Comment by rogery555** [comment]
Yes it would be helpful for some extra facial illumination when vlogging.   Also for use with my macro lens,  that screen could add plenty of light for shooting closeup objects without the need to deal with an external light.

**Comment by rogery555** [comment]
@dnewman-gpsw  So what do you think, how likely would it be that I could ever see this option available on my Gopro 9?

**Comment by Little9070** [comment]
It would be good for scanning a QR code in the dark.
